podium-gateway/src/main/java/nl/thehyve/podium/config/MicroserviceSecurityConfiguration.java

```diff
@@ -39,10 +41,13 @@
 @EnableWebSecurity
 public class MicroserviceSecurityConfiguration extends ResourceServerConfigurerAdapter {
 
+    private final Logger log = LoggerFactory.getLogger(MicroserviceSecurityConfiguration.class);
+
     private final PodiumProperties podiumProperties;
 
     private final DiscoveryClient discoveryClient;
 
+    @Inject
```

Comment:
I wont complain :), but made a ticket for this [PODIUM-93](https://jira.thehyve.nl/browse/PODIUM-93)